data/coalition/coalition missions.txt

```diff
@@ -893,9 +893,9 @@ mission "Kimek Gift"
 		payment 36750
 		"coalition jobs" += 2
 		conversation
-			`Apparently this Saryd in question doesn't live here in the spaceport village, so you must take a train to go to his home. Thankfully, a train ticket was attached to the package, so you waste no time and get on the next train there.`
+			`Apparently the Saryd in question doesn't live here in the spaceport village, so you must take a train to go to his home. Thankfully, a train ticket was attached to the package, so you waste no time and get on the next train there.`
 			`	Even for a world without too large a population, the train wagons here feel unusually empty, with only a handful of other passengers on the one you settle in. The trip takes a few hours, so you have plenty of time to look out the window and gaze at the sights of the seemingly infinite winter forest.`
-			`	Halfway through, a blizzard makes it tough to see much outside. The ticket inspectors come through again, but now they are serving something akin to hot chocolate, only it is instead a blue, creamy substance.`
+			`	Halfway through, a blizzard makes it tough to see much outside. The ticket inspectors come through again, but now they are serving something akin to hot chocolate - only it is instead a blue, creamy substance.`
```

Comment:
Technically this was fine before. I'm not sure adding a dash actually fixes the overuse of commas?
Maybe a rewrite entirely (though this may not be the exact way to do it):
```suggestion
			`	Halfway through, a blizzard makes it tough to see much outside. The ticket inspectors come through again, now serving something akin to hot chocolate, although this has instead a blue creamy color.`
```

Reply:
Yeah, I didn't quite know how to work with that sentence. I don't think this is much better, though. How about:
> [...] come through again, now serving something akin to hot chocolate - if the hot chocolate was instead a blue, creamy substance.

Reply:
Maybe 
"[...] come through again, now serving a creamy blue drink that tastes similar to hot chocolate"

Unless they mean that it is similar in function to hot chocolate. In that case perhaps:
"[...] come through again, now serving a creamy blue drink which [appears to be / must be] their equivalent of hot chocolate."

Reply:
I'll take the second, but with "seems to" instead.